src/app/support/page.tsx

```diff
@@ -47,28 +47,29 @@ export default function SupportPage() {
             </SiteLabel>
             {supportPageInfo.goalDetail}
           </InfoBox>
+          <div className="EstimatedTimelineContainer mx-auto flex w-full flex-col self-end pb-10 pt-12">
+            <h1 className="EstimatedTimelineTitle self-end pb-8">
+              our est. timeline:
+            </h1>
+            <InfoBox
+              aria="our est. timeline"
+              variant="hollow"
+              addClasses="text-sm py-8 px-12 self-end"
+            >
+              {supportPageInfo.estTimeline.map((status, index) => (
+                <p key={index} className="mb-2 last:mb-0">
+                  {status}
+                </p>
+              ))}
+            </InfoBox>
+            <p className="TimelineNote mt-8 max-w-md self-end text-right text-xs font-medium italic text-olive">
+              {supportPageInfo.timelineAst}
+            </p>
+          </div>
         </div>
         <DonationBox />
       </div>
-      <div className="EstimatedTimelineContainer mx-auto flex w-full flex-col self-end border-b-2 border-midnight/20 pb-10">
-        <h1 className="EstimatedTimelineTitle mt-12 self-end">
-          our est. timeline:
-        </h1>
-        <InfoBox
-          aria="our est. timeline"
-          variant="hollow"
-          addClasses="text-sm py-8 px-12 self-end"
-        >
-          {supportPageInfo.estTimeline.map((status, index) => (
-            <p key={index} className="mb-2 last:mb-0">
-              {status}
-            </p>
-          ))}
-        </InfoBox>
-        <p className="TimelineNote mt-8 max-w-md self-end text-right text-xs font-medium italic text-olive">
-          {supportPageInfo.timelineAst}
-        </p>
-      </div>
+
       <CrowdfundingRewardsSection />
```

Comment:
Importing CrowdfundingRewardsSection but leaving the first section of this page declared in page is confusing and inconsistent. It would be better to extract the Donation box into its own file, that's imported into the top section, which is imported into the page.tsx